help/print-a-conversation.md

```diff
@@ -0,0 +1,45 @@
+# Print a conversation
+
+Conversations, including topics, DMs, and group DMs, are printable from the
+Zulip web app.
```

Comment:
As noted in #26397, we should probably add a link here to clarify meaning. I would also tweak the intro to something like:

> Zulip [conversations](/help/recent-conversations), including streams, topics, and direct messages, are printable from the Zulip web app.

or more concisely:

> Zulip [conversations](/help/recent-conversations) are printable from the Zulip web app.